src/Service/ElasticsearchServiceMock.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Contains a mock of ElasticsearchService.
+ */
+
+namespace App\Service;
+
+/**
+ * Class ElasticsearchService.
+ */
+class ElasticsearchServiceMock extends ElasticsearchService
```

Comment:
Is this mock service needed? If so I don't think it should be places with the normal services code or it should be placed inside test folder and namespaced under test.

Reply:
Look at https://symfony.com/doc/4.4/create_framework/unit_testing.html and search for `createMock`

Reply:
Fixed